Pomodoro.Services/StatisticsService.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Pomodoro.Core.Enums;
+using Pomodoro.Core.Interfaces.IServices;
+using Pomodoro.Core.Models.Statistics;
+using Pomodoro.DataAccess.EF;
+using Pomodoro.DataAccess.Entities;
+
+namespace Pomodoro.Services
+{
+	public  class StatisticsService : IStatisticsService
+	{
+		public async Task<DailyStatistics> GetDailyStatisticsAsync(Guid userId, DateOnly day)
+		{
+			DailyStatistics dailyStatistics = new DailyStatistics();
+
+			using (AppDbContext db = new AppDbContext())
```

Comment:
As I understand our architecture we should use repositories in our services, not DbContext directly

Reply:
@KonstantinZh take a look at the SettingsService in #17 PR as an example.